internal/bminventory/inventory.go

```diff
@@ -237,6 +261,96 @@ const nodeIgnitionFormat = `{
   }
 }`
 
+const configStaticIpsScript = `
+#!/bin/bash
+
+function create_template_file() {
+    cat > "/var/tmp/template.connection" <<EOF
+[connection]
+id=\$FOUND_INTERFACE
+interface-name=\$FOUND_INTERFACE
+type=ethernet
+multi-connect=3
+autoconnect=true
+autoconnect-priority=1
+
+[ethernet]
+mac-address-blacklist=
+
+[ipv4]
+method=manual
+addr-gen-mode=eui64
+addresses=\$FOUND_IP/\$FOUND_MASK
+gateway=\$FOUND_GW
+dns=\$FOUND_DNS
+
+[ipv6]
+method=auto
+addr-gen-mode=eui64
+
+[802-3-ethernet]
+mac-address=\$FOUND_MAC
+EOF
+}
+
+function find_my_mac() {
+    MAC_TO_CHECK=${1}
+
+    for entry in $(cat "/var/tmp/static_ips_config.csv")
```

Comment:
csv? :)

Reply:
why not ? :)

Reply:
We tought that a CSV is an easy way to move from a inventory of IPs to be parsed by a script. Our Customer 0 Had an inventory of IPs in this way (not with the same format, but something similar)